examples/cms-builder-io/tailwind.config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  purge: ['./components/**/*.js', './pages/**/*.js'],
```

Comment:
Which includes using JIT (see other CMS examples 😄 )

Reply:
added in the line above https://github.com/vercel/next.js/pull/22094/files#diff-0baabb040d16c756b4cf9551bb9b391fb338099b8fc539557723a0ad082f884bR2

Reply:
Could we update to Tailwind 3? It removes `mode` entirely, JIT is the default, and changes `purge` to `content`.

Reply:
@leerob done